modules/core/src/v2/coins/ttrx.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Testnet TRX - Shasta network
+ *
+ * @format
+ */
+import { BaseCoin } from '../baseCoin';
+import { Trx } from './trx';
+
+export class Ttrx extends Trx {
+  protected constructor(bitgo: any) {
+    super(bitgo);
+  }
+
+  static createInstance(bitgo: any): BaseCoin {
+    return new Ttrx(bitgo);
+  }
+  /**
+   * Identifier for the blockchain which supports this coin
+   */
+  public getChain(): string {
+    return 'ttrx';
+  }
+
+  /**
+   * Complete human-readable name of this coin
+   */
+  public getFullName(): string {
+    return 'Testnet Tron (Shasta)';
```

Comment:
What is shasta?

Reply:
It's the most popular Tron private test network: https://www.trongrid.io/shasta

Reply:
I'll add a comment to this line

Reply:
It's a bit strange though. For example, teth is just "Testnet Ethereum", not "Testnet Ethereum (Kovan)"

Reply:
I'm pulling from statics now